modules/images/webp-uploads/image-edit.php

```diff
@@ -420,3 +420,26 @@ function webp_uploads_restore_image( $attachment_id, $data ) {
 
 	return $data;
 }
+
+/**
+ * Compatibility function to check whether editing image thumbnails separately is enabled.
+ *
+ * The filter {@see 'image_edit_thumbnails_separately'} was introduced in WordPress 6.3 with default value of `false`,
+ * for a behavior that previously was always enabled.
+ *
+ * @since n.e.x.t
+ * @see https://core.trac.wordpress.org/ticket/57685
+ *
+ * @return bool True if editing image thumbnails is enabled, false otherwise.
+ */
+function webp_uploads_image_edit_thumbnails_separately() {
+	if ( version_compare( get_bloginfo( 'version' ), '6.3', '<' ) ) {
+		return true;
+	}
+
+	/*
+	 * This filter was introduced in WordPress 6.3 with default value `false`,
```

Comment:
This is already stated in the function docblock above, so a bit redundant. We shoukd rather follow the coding standards to say „This filter was documentend in xyz…“ (don‘t know the exact wording right now)